pkg/util/rpauthorizer/rpauthorizer.go

```diff
@@ -21,12 +21,17 @@ type RPAuthorizer interface {
 type devRPAuthorizer struct{}
 
 func (devRPAuthorizer) NewRPAuthorizer(resource string) (autorest.Authorizer, error) {
+	im, err := instancemetadata.NewDev()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
don't do this - pass in the instancemetadata via rpauthorizer.New()

Reply:
changed